crates/transaction-generator-lib/src/publishing/module_simple.rs

```diff
@@ -395,6 +436,7 @@ impl EntryPoints {
             EntryPoints::Nop2Signers => MultiSigConfig::Random(1),
             EntryPoints::Nop5Signers => MultiSigConfig::Random(4),
             EntryPoints::TokenV2AmbassadorMint => MultiSigConfig::Publisher,
+            EntryPoints::ResourceGroupsGlobalResource => MultiSigConfig::Publisher,
```

Comment:
What about ResourceGroupIndividualResource and ResourceGroupMutichange variants?